.github/workflows/trestlebot-create-component-definition.yml

```diff
@@ -0,0 +1,53 @@
+name: Trestle-bot Create Component Definition
+
+on:
+  workflow_dispatch:
+   inputs:
+      profile_name:
+        description: Name of the Trestle profile to use for the component definition
+        required: true
+      component_definition_name:
+        description: Name of the component definition to create
+        required: true
+      component_title:
+        description: Name of the component to create in the generated component definition
+        required: true
+      component_type:
+        description: Type of the component (e.g. service, policy, physical, validation, etc.)
+        required: false
+        default: "service"
+      component_description:
+        description: Description of the component to create
+        required: true
+
+jobs:
+  create-component-definition:
+    name: Create component definition
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Bump this to v4? Maybe on a subsequent PR we could add a dependabot config for GH actions.